.github/workflows/test.yml

```diff
@@ -19,10 +19,18 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-      - name: build and test
+      - name: build
         run: |
           yarn --ignore-engines
           yarn build
-          yarn test
+        env:
+          CI: true
+      - name: test
+        run: npm test
+        env:
+          CI: true
+      - name: dev test
+        run: npm run test:dev
+        if: matrix.os == 'ubuntu-latest'
```

Comment:
this will run on Node versions, but only on Ubuntu. I think that makes sense, but let me know if you only want to test on a single node version

Reply:
We eventually want to run this on all relevent node versions + Ubuntu & Windows, so the closer to that we start, the better.

Glad that you were able to get this into a mergeable state!